### ISSUE TITLE
Open API collection import doesn't show path parameters with query parameters

### ISSUE THREAD

**Issue by jjspace** [rangav/thunder-client-support#907]
**Describe the bug**
When importing a collection from an openapi yaml file it doesn't seem to import both path and query parameters. Routes that use both only show the query parameters whereas routes with only path params are correctly including them in the list. 

**To Reproduce**

Minimal example int he expand section below. Create a collection with this file and check the routes created. Observe that the `GET List available posts` route doesn't include the `userId` as a parameter under the query tab.

Note if you modify the route at all it will completely replace all the query params with the one `userId` path param but then you can't use the other query params that were already set up

<details>
  <summary>Expand for example</summary>

```yaml
openapi: 3.0.2

info:
  title: Test
  version: v1
paths:
  /v1/users/{userId}/posts:
    parameters:
      - $ref: "#/components/parameters/userId"
    get:
      operationId: getPosts
      summary: List available posts
      description: |
        Retrieves all posts
      tags:
        - Posts
      parameters:
        - in: query
          name: createdBy
          schema:
            type: array
            items:
              $ref: "#/components/schemas/UserIdentifier"
            description: Return posts created by any of the provided user ids. Passed as comma separated list.
            example: 1381e588-4f48-493d-a24c-c83db6f6602b,65cd8803-3c9d-4329-a006-0b83d8453ff5
        - in: query
          name: modifiedAfter
          schema:
            type: string
            format: date-time
            description: Return posts modified after specified date.
            example: 2022-03-04T15:16:34.000Z
        - in: query
          name: modifiedBefore
          schema:
            type: string
            format: date-time
            description: Return posts modified before specified date.
            example: 2022-03-04T15:16:34.000Z
        - in: query
          name: offset
          schema:
            type: number
            description: Offset into list of posts. When combined with the limit parameter this enables paging.
            minimum: 0
        - in: query
          name: limit
          schema:
            type: number
            description: Number of posts to return. When combined with the offset parameter this enables paging.  The server may return less than the requested amount.
            maximum: 100
        - in: query
          name: sortOrder
          schema:
            allOf:
              - $ref: "#/components/schemas/SortOrder"
              - default: DESC
              - description: Order to sort date modified.
      responses:
        200:
          description: OK
          headers:
            link:
              schema:
                type: string
          content:
            application/json:
              schema:
                title: Posts
                type: object
                properties:
                  total:
                    type: integer
                    description: The total amount of items available.
                  items:
                    type: array
                    description: A page of posts
                    items:
                      $ref: "#/components/schemas/Post"
              examples:
                exampleResponse:
                  value:
                    {}
        401:
          $ref: "#/components/responses/401"
  /v1/users/{userId}/posts/{postId}:
    parameters:
      - $ref: "#/components/parameters/userId"
      - $ref: "#/components/parameters/postId"
    get:
      operationId: getPost
      summary: Get post details
      description: |
        Retrieves all information about a single post.
      tags:
        - Posts
      security:
        - Dashboard: []
      responses:
        200:
          description: OK
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Post"
              examples:
                exampleResponse:
                  value:
                    {}
        401:
          $ref: "#/components/responses/401"
components:
  parameters:
    userId:
      description: A user identifier.
      in: path
      name: userId
      schema:
        $ref: "#/components/schemas/UserIdentifier"
      required: true
    postId:
      description: A post identifier.
      in: path
      name: postId
      schema:
        $ref: "#/components/schemas/PostIdentifier"
      required: true
  schemas:
    UserIdentifier:
      type: string
      format: uuid
      description: The unique identifier for a user.
      example: 04df70ad-459d-47fc-bbc3-0a9df534c76a
    PostIdentifier:
      type: integer
      description: The unique identifier of this post.
      example: 102
    Post:
      type: object
      allOf:
        - type: object
          properties:
            id:
              $ref: "#/components/schemas/PostIdentifier"
        - type: object
          properties:
            createdBy:
              $ref: "#/components/schemas/UserIdentifier"
            dateCreated:
              type: string
              format: date-time
              description: Time the post was created in ISO 8601 format
            dateModified:
              type: string
              format: date-time
              description: Time the post was changed in ISO 8601 format
            modifiedBy:
              $ref: "#/components/schemas/UserIdentifier"
    SortOrder:
      type: string
      enum:
            - ASC
            - DESC
  responses:
    401:
      description: Invalid Credentials
    404:
      description: Resource Not Found
```

</details>

**Expected behavior**
I expect both the query params _and_ path params to be shown in the list so it's easy to set them all

**Platform:** 
 - OS: Kubuntu 20.04
 - vscode version: `1.73.1`
 - extension version: `v2.0.2`

**Comment by rangav** [comment]
Thanks @jjspace for reporting the bug, will fix it asap.

**Comment by rangav** [comment]
This Bug is fixed is now and published to marketplace, please update to `v2.1.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.1.0

Please let me know your feedback.

**Comment by rangav** [comment]
@jjspace thanks for confirmation 👍 , let me know if you have any questions/issues.
